dkc/core/templates/core/staff_tree_size.html

```diff
@@ -0,0 +1,38 @@
+{% extends 'core/staff_base.html' %}
+
+{% block extra_head %}
+<style type="text/css">
+  table th {
+    border: 1px solid black;
+}
+</style>
+{% endblock %}
+
+{% block body_content %}
+<h1>Tree Sizes</h1>
+
+<table>
```

Comment:
Does Tailwind not provide us some nice table styling we could use?

Reply:
#181